DEVELOPING.md

```diff
@@ -33,9 +33,10 @@ If you have any questions, feel free to ask on the issue or join us on [Slack](h
 ### Installation
 
 1. Install [NodeJS](https://nodejs.org).
-2. In the repo directory, run `npm i` command to install the required npm packages.
-3. Run `npm i -g gulp` command to install gulp system-wide (on Mac or Linux you may need to prefix this with `sudo`, depending on how Node was installed).
-4. Edit your hosts file (`/etc/hosts` on Mac or Linux, `%SystemRoot%\System32\drivers\etc\hosts` on Windows) and map `ads.localhost` and `iframe.localhost` to `127.0.0.1`.
+2. Install [yarn](https://yarnpkg.com/en/docs/install) (yarn is faster and gives us deterministic builds)
```

Comment:
"yarn is faster" than what?  npm?  you may not need that line if you're removing the reference to npm.

For the end-to-end guide I'm writing, does this mean that we aren't using npm any more at all?  Or is it still used under the covers?